### PR TITLE
Fixing Issue with latest merge of Jerry to main #13

### LINKED ISSUE #13: Issue with latest merge of Jerry to Main
I followed the convention of:
.backend
.quickbytes.backend.controller
.quickbytes.backend.model
.quickbytes.backend.repository
.quickbytes.backend.service

As shown here:
image

While the original repository followed the following structure:
.quickbytes.controller
.quickbytes.model
.quickbytes.repository

As shown here:
image

### PR DESCRIPTION
**Pull request by sshJerry**
Following original repository structure of:
.quickbytes.controller
.quickbytes.model
.quickbytes.repository

### CODE REVIEW

**Review by jayharris14 — APPROVED** [review]

File: QuickBytes/src/main/java/com/quickbytes/controller/CustomerController.java [hunk 1]
```diff
@@ -1,4 +1,4 @@
-package com.quickbytes.controller;
+package com.quickbytes.backend.controller;
```
Comment (Kev-Index) [inline QuickBytes/src/main/java/com/quickbytes/controller/CustomerController.java]: All the package names should be without the ".backend"

File: README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # SpingQuick
```
Comment (Kev-Index) [inline README.md]: We can remove "# SpingQuick" and just keep "# QuickBytesOnSpring" for a single title

File: QuickBytes/src/main/java/com/quickbytes/controller/CustomerController.java (hunk 1 above)
Comment (sshJerry) [reply QuickBytes/src/main/java/com/quickbytes/controller/CustomerController.java]: Resolved

File: README.md (hunk 2 above)
Comment (sshJerry) [reply README.md]: Resolved

**Comment by sshJerry** [comment]
Resolved #13 alongside the slight adjustment to readme

**Pull request merged**